analysis/import/SonarImporter/src/main/java/de/maibornwolff/codecharta/importer/sonar/model/Metrics.java

```diff
@@ -37,7 +37,15 @@
 public class Metrics {
     private List<MetricObject> metrics;
 
+    private int total;
+
     public List<MetricObject> getMetrics() {
```

Comment:
is Metrics supposed to be an immutable object? then don't return List. If not it still makes sense to return something more generic, perhaps, Iterable or Collection instead of List